common/log.cpp

```diff
@@ -76,4 +79,7 @@ void yamiTraceInit()
         }
         isInit = 1;
     }
+#ifdef __ENABLE_DEBUG__
+     fprintf(stderr, "yami log isn't enabled (--enable-debug)\n");
```

Comment:
not catch your meaning, do you mean to below setence?

 ERROR("yami log isn't enabled (--enable-debug)\n");